src/lib/components/empty.svelte

```diff
@@ -40,12 +41,13 @@
             </button>
             <slot>
                 <div class="u-text-center">
-                    <Heading size="7" tag="h2">Create your first {target} to get started.</Heading>
+                    <Heading size="7" tag="h2" trimmed={false}
+                        >Create your first {target} to get started.</Heading>
```

Comment:
Nitpick, can we fix this so the `>` doesn't go to a new line? :laughing:

Reply:
Done!